src/api/roleGrants.ts

```diff
@@ -0,0 +1,15 @@
+import { insertSupabase, TABLES } from 'services/supabase';
+import { Capability } from 'types';
+
+export const createRoleGrant = (
+    subject_role: string,
+    object_role: string,
+    capability: Capability
+) => {
+    return insertSupabase(TABLES.ROLE_GRANTS, {
+        subject_role,
+        object_role,
+        capability,
+        detail: 'Applied via UI',
```

Comment:
Did we review this with folks? I know we have some details auto filled in and previously we had messages like this but decided to remove them as for audit purposes having "by the CLI" and blank was good enough.